src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -83,6 +83,7 @@ export type HostConfig<T, P, I, TI, PI, C, CX, PL> = {
 
   shouldSetTextContent(props : P) : boolean,
   resetTextContent(instance : I) : void,
+  areChildrenOffscreen(props : P) : boolean,
```

Comment:
Maybe we should pass the `type` also? Sometimes that's useful. Maybe we need that for `shouldSetTextContent` too.

`areChildrenOffscreen` is also present tense but this is really referring to if something should become offscreen.